manifests/globals.pp

```diff
@@ -94,7 +94,13 @@
     'FreeBSD' => '93',
     'OpenBSD' => '9.3',
     'Suse' => $::operatingsystem ? {
-      'SLES' => '91',
+      'SLES' => $::operatingsystemrelease ? {
+        '11.3' => '91',
```

Comment:
the `default => '91'` would be wrong though eg on SLES11SP2 (and maybe on the upcoming SLES12SP1, I need to check it). How about `default => 'undef'`?